composer/algorithms/fused_layernorm/fused_layernorm.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2022 MosaicML Composer authors
+# SPDX-License-Identifier: Apache-2.0
+
+# Copyright 2022 MosaicML. All Rights Reserved.
+
+from __future__ import annotations
+
+import logging
+from typing import Optional, Sequence, Union
+
+import torch
+
+try:
+    from apex.normalization.fused_layer_norm import FusedLayerNorm as APEXFusedLayerNorm
```

Comment:
Since we import all algos, this will always trigger, can we do a:

``
try:
   import apex
except ImportError
APEX_INSTALLED = False
```

and then trigger the failures if apply_fused_layernorm is called?

Reply:
Ah, good call. Can do!